api/http/server.go

```diff
@@ -285,6 +292,16 @@ func (s *Server) HandlePostRaw(w http.ResponseWriter, r *http.Request) {
 
 	log.Debug("stored content", "ruid", ruid, "key", addr)
 
+	// Add the root hash of the RAW file in the pinFilesIndex
+	if headerPin != "" {
```

Comment:
True is set as just a place holder because the header items are KV. The absence indicated that pinning is not needed. If this is confusing, i can set the value to "".

Reply:
Fixed it. Now checking for false too.

**Note**: Several of these are taken from tags implementation. Tags also checks only for not empty and not for false in its header.  Another example is "SwarmTagHeaderName" ... where "Swarm" is  not required according to a comment i got.  May be we should also fix them too.